### PR TITLE
Added initial release notes for v. 1.0.0

### PR DESCRIPTION
**Pull request by susanmdouglas-aws**
Signed-off-by: susanmdouglas <susandou@amazon.com>

### Description
Added initial release notes for Babelfish 1.0.0.
 
### Issues Resolved
Added initial release notes for Babelfish 1.0.0.

### Check List
- [ x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by rcv-aws — APPROVED** [review]

File: _releases/release_notes_1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Release Notes 1.0.0
+nav_order: 2
+---
+
+Babelfish Version: 1.0.0
+PostgreSQL Server Version: 13.2
+Minimum Compass Version: [1.0.0]](https://github.com/babelfish-for-postgresql/babelfish_compass/)
```
Comment (3manuek) [inline _releases/release_notes_1.0.0.md:9]: For Compass release we can use the release link: https://github.com/babelfish-for-postgresql/babelfish_compass/releases/tag/v1.0

File: _releases/release_notes_1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Release Notes 1.0.0
+nav_order: 2
+---
+
+Babelfish Version: 1.0.0
```
Comment (3manuek) [inline _releases/release_notes_1.0.0.md:7]: Once release files are ready, I'll provide the link to the 1.0.0 release.

File: _releases/release_notes_1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+PostgreSQL Server Version: 13.2
```
Comment (3manuek) [inline _releases/release_notes_1.0.0.md:8]: Babelfish 1.0.0 is on top of 13.4.

File: _releases/release_notes_1.0.0.md (hunk 1 above)
Comment (susanmdouglas-aws) [reply _releases/release_notes_1.0.0.md:9]: Added....

File: _releases/release_notes_1.0.0.md (hunk 2 above)
Comment (susanmdouglas-aws) [reply _releases/release_notes_1.0.0.md:7]: Sounds good...

**Review by 3manuek — APPROVED** [review]

**Review by nasbyj — APPROVED** [review]

**Pull request merged**